src/color.coffee

```diff
@@ -0,0 +1,46 @@
+_ = require 'underscore-plus'
+ParsedColor = require 'color'
+
+# Public: A simple Color class returned from `atom.config.get` when the value at
```

Comment:
Whoops, forgot that was the new jam, yeah, I could go either way, but Essential sounds good.